csharp/src/Google.Protobuf/UnknownField.cs

```diff
@@ -64,6 +64,8 @@ public UnknownField()
         {
         }
 
+        internal List<ByteString> LengthDelimitedList => lengthDelimitedList;
```

Comment:
I'd like to get away without exposing lengthDelimitedList (it's the only one we're exposing and feels odd).

Reply:
If we expose the rest of the lists would that be ok? I don't know how I feel about adding a function to UnknownField just to create a message value from the length delimited values in it.